common/changes/@rushstack/terminal/move-terminal_2022-01-20-19-34.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/terminal",
+      "comment": "Introduce a Terminal, Colors, AsciEscape, and some related APIs. These APIs were previously in the @rushstack/node-core-library package.",
```

Comment:
We should have a note somewhere (maybe just your PR description?) that details exactly which APIs were changed with sufficient detail to facilitate easy migration. (See [this report](https://octogonz.github.io/files/pr3176-diff.htm) for help.) Something like:

- `node-core-library/Colors` and `node-core-library/IColorableSequence` from `` were removed; instead use the new `terminal/Colorize`
- `node-core-library/ITerminalWritableOptions` was renamed to `terminal/ITerminalStreamWritableOptions`
- etc...

Then the CHANGELOG.md for both NCL and Terminal should link to this note.